packages/jellyfish-testing/src/testingwrapper.ts

```diff
@@ -0,0 +1,32 @@
+import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { ContainerGroup, DeFiDContainer, MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing, TestingGroup } from '.'
+
+export class TestingWrapper {
+  create (): Testing
+  create (n: number): TestingGroup
+  create (n: number, init: (index: number) => DeFiDContainer): Testing | TestingGroup
```

Comment:
```suggestion
  create (): Testing
  create (n: 1): Testing
  create (n: number): TestingGroup
  create (n: 1, init: (index: number) => DeFiDContainer): Testing
  create (n: number, init: (index: number) => DeFiDContainer): Testing | TestingGroup
```
If we add case for `1` input we are addressing the case on line 15 allowing us to avoid having to cast to `Testing` in our unit tests.

Reply:
lovely! thanks. i like this one